src/screens/CreatePost/index.js

```diff
@@ -449,11 +450,13 @@ const CreatePost = () => {
   };
 
   const postV2 = async () => {
+    setLoadingPost(true);
     if (!isEmptyMessageAllowed()) {
       showMessage({
         message: StringConstant.createPostFailedNoMessage,
         type: 'danger'
       });
+      setLoadingPost(false);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This block of code handles the case where an empty message is not allowed. It sets the `loadingPost` state to true at the start and false after showing the error message. This is a good practice as it provides immediate feedback to the user about the ongoing process and any errors that occur.
```diff
+    setLoadingPost(true);
     if (!isEmptyMessageAllowed()) {
       showMessage({
         message: StringConstant.createPostFailedNoMessage,
         type: 'danger'
       });
+      setLoadingPost(false);
       return true;
     }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->